server/migrations/env.py

```diff
@@ -1,7 +1,9 @@
 from __future__ import with_statement
 from alembic import context
-from sqlalchemy import engine_from_config, pool
 from logging.config import fileConfig
+from sqlalchemy import engine_from_config, pool, MetaData
+from server.models import category, event_history, event_status, event_tag, event, feedback, gallery, role, subscribe, tag, user_status, user, token
```

Comment:
According to [PEP-8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length):
> Limit all lines to a maximum of 79 characters.